src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -51,20 +51,27 @@ public Exercise(int age) {
         Create a constructor that accepts both a String and an int as parameters, in that order, and assign the values
         provided to the name and age members
      */
-
+    public Exercise(String name, int age){
+        this.name=name;
+        this.age=age;
+    }
 
 
     /*
         2. Create a method named add that accepts two integers. The method should return the numbers added together.
      */
-
+    public int add (int a, int b){
+        return a+b;
+    }
 
 
     /*
         3. Create another method named add that accepts two Strings. The method should return the strings concatenated
         together with a space in between.
      */
-
+    public String add(String a,String b){
+        return a+" "+b;
```

Comment:
best practice to leave spaces for readability, eg: 

`public String add(String a, String b){
        return a + " " + b;
        ...`